feel/feelcrb/crb.hpp

```diff
@@ -1385,7 +1385,7 @@ class CRB : public CRBDB
     scm_ptrtype M_scmM;
 
     //export
-    export_ptrtype M_exporter;
+    mutable export_ptrtype M_exporter;
```

Comment:
>in crb.hpp, the method exportBasisFunctions is marked const but if M_exporter is not yet initialized, it will try to initialize it, which will not compile.
There is two ways to fix this, remove the const of the function, or make M_exporter mutable.
What should we do ?

I asked that on gitter, but apparently, it will cause a conflict because @jbwahl removed const in his branch.